core/compatibility/plugins/config/index.js

```diff
@@ -1,5 +1,5 @@
-import store from '@vue-storefront/core/store'
-const config = store.state.config
+import config from 'config'
```

Comment:
in some places you use `globalConfig` and in some `config` i think we should just have named export instead of default one so the naming is consistant everywhere. It's generally recomended to use named exports

Reply:
I don't think it provides us any value and requires a lot of changes in the docs etc. Can't apply this one.